CHANGELOG.md

```
@@ -1,3 +1,17 @@
+# [6.1.0](https://github.com/equationalapplications/expo-llm-wiki/compare/v6.0.1...v6.1.0) (2026-08-30)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)